src/index.ts

```diff
@@ -69,20 +69,47 @@ function isVisible(n: any) {
   return n.visible && n.opacity > 0.001 && !n.removed;
 }
 
-function conditionalReadBlacklist(n: any) {
+interface Options {
+  skipInvisibleNodes: boolean;
+  geometry: "none" | "paths";
+}
+
+const defaultOptions: Options = {
+  skipInvisibleNodes: true,
+  geometry: "none"
+};
+
+function conditionalReadBlacklist(n: any, options: Pick<Options, "geometry">) {
+  // Ignore geometry keys if geometry is set to "none"
+  // Copied these keys from the Figma REST API.
+  // "size" represents width/height of elements and is different
+  // from the width/height of the bounding box:
+  // https://www.figma.com/developers/api#frame-props
+  if (options.geometry === "none") {
+    return new Set([
+      ...readBlacklist,
+      "fillGeometry",
+      "strokeGeometry",
+      "size",
+      "relativeTransform"
+    ]);
+  }
+
+  // Never include text outline geometry
   if ("type" in n && n.type === "TEXT") {
     return new Set([...readBlacklist, "fillGeometry", "strokeGeometry"]);
```

Comment:
slightly unclear that this is a superset of TEXT below. ok for now, will have to change in my work